docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -878,10 +863,11 @@ For example, the following payload disables TSDS on `nginx.stubstatus`:
 [[data-streams-advanced-synthetic-enable]]
 == Enable synthetic `_source`
 
+Note that synthetic `_source` is supported only on non-TSDS data streams.
```

Comment:
Synthetic source is also supported on tsds data streams. I think this line can be removed?

Reply:
Thanks @martijnvg. I can remove the line, but from @lucabelluccini's [issue](https://github.com/elastic/ingest-docs/issues/1618): `I've tested it on 8.17 and it's impossible to disable Synthetic _source on a TSDS data stream.`

For now I've changed it to `Note that disabling synthetic _source is supported only on non-TSDS data streams.`
Please let me know if that's not what we want.

Reply:
I commented on the issue: https://github.com/elastic/ingest-docs/issues/1618#issuecomment-2602660156

Let's see what went wrong here.